pkg/sql/logic_test.go

```diff
@@ -454,6 +454,12 @@ type logicQuery struct {
 	// expectedErr and expectedErrCode are as in logicStatement.
 	expectErr     string
 	expectErrCode string
+
+	// if set, the results are cross-checked against previous queries with the
+	// same label.
+	label string
```

Comment:
Hmm, I think you're right. Maybe we aren't actually cross-checking the labels, I'll investigate and fix.
Yes, it's used to generate an expected result instead of hardcoding it, see https://github.com/cockroachdb/cockroach/pull/14543/commits/9a92f9943863ab993c9c0336fe5efdcd9efd560a#diff-9f1da1aa51f288bda7cd06242aa191a3R186 (line 140)